src/coreclr/jit/emitxarch.cpp

```diff
@@ -7464,6 +7464,9 @@ void emitter::emitIns_R_C(
         else
 #endif // TARGET_X86
         {
+            SetEvexBroadcastIfNeeded(id, instOptions);
+            SetEvexEmbMaskIfNeeded(id, instOptions);
```

Comment:
We have 37 instances where we `SetEvex` information, and this is the only place where it was happening in the wrong spot and is what was causing the size misprediction